packages/engine/lib/nano/src/lib.rs

```diff
@@ -42,15 +41,6 @@
     clippy::missing_inline_in_public_items,
     reason = "We didn't any serious optimization work until now"
 )]
-#![allow(
```

Comment:
Are these all of the restriction lints or have we lost some?

Reply:
These were only allows for specific restriction lints.

Reply:
Yes, right. I chose that restriction lints were way too restrictive (things like `pub use` is warned). Clippy suggests to only use specific lints from `restriction`, not the whole category.

Reply:
Alright, would have been good to get that reasoning 😄

Reply:
Yes, sorry 🙃